js/project-page-functions.js

```diff
@@ -112,8 +112,14 @@ function onProjectPageLoaded() {
     }
   });
   var $body = $('body');
-  $body.scrollspy({target: '#right-sidebar-nav', offset: navHeight});
-  $body.scrollspy({target: '#left-sidebar-nav', offset: navHeight});
+  $body.scrollspy({'target': '#right-sidebar-nav', 'offset':navHeight});
+  // Offset in the above for some reason doesn't work, so we fix it this way with a little hack:
```

Comment:
You removed the offset for the `#left-sidebar-nav` and did not replace it with anything. Was this intentional?